js/webui.js

```diff
@@ -123,6 +123,7 @@ var theWebUI = {
 		"webui.confirm_when_deleting":	1,
 		"webui.alternate_color":	0,
 		"webui.side_panel_min_width": 	200,
+		"webui.side_panel_max_width_percent": 50,
```

Comment:
We can set this to a more sane value like `35`. It's not going to impact mobile devices. We can assume 720p or higher screen.